### PR TITLE
User can choose to install missing dependencies

### LINKED ISSUE #70: Allow users to choose the auto-install
Ideally we'll show the list of uninstalled packages to the user and ask them if they want to install it, or if not now or not at all

### PR DESCRIPTION
**Pull request by archcorsair**
Prompts a user to choose if they would like to install dependencies for a package that requires it.
Closes #70

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -14,14 +14,43 @@ if (typeof window.__steelbrain_package_deps === 'undefined') {
 async function installDependencies(packageName: ?string): Promise<void> {
   invariant(packageName, '[Package-Deps] Failed to determine package name')
 
-  const dependencies = Helpers.getDependencies(packageName)
+  let dependencies = Helpers.getDependencies(packageName)
```
Comment (steelbrain) [inline src/index.js]: s/let/const/

File: src/index.js [hunk 2]
```diff
@@ -14,14 +14,43 @@ if (typeof window.__steelbrain_package_deps === 'undefined') {
...
+
+  if (!userAccepted) {
+    console.log(`User rejected installation of ${packageName} dependencies`)
+    return
+  }
+
   const view = new View(packageName, dependencies)
   const errors = await Helpers.apmInstall(dependencies, function() {
-    view.advance()
+    resolve(view.advance())
```
Comment (steelbrain) [inline src/index.js]: Also remove this resolve here

File: src/index.js (hunk 1 above)
Comment (archcorsair) [inline src/index.js]: fixed

File: src/index.js (hunk 2 above)
Comment (archcorsair) [inline src/index.js]: fixed

**Comment by steelbrain** [comment]
Thanks @archcorsair! 🙇

**Pull request merged**